infinite_tracing/test/infinite_tracing/client_test.rb

```diff
@@ -114,9 +120,11 @@ def test_handles_suspended_state
             connection.stubs(:retry_connection_period).returns(0)
 
             total_spans = 5
-            emulate_streaming_segments total_spans do |client, segments|
+            _spans, _segments = emulate_streaming_segments total_spans do |client, segments|
```

Comment:
If you don't need _either_ response object (marking them as ignored with the underscore prefix), can you just drop everything up to and including the equals sign entirely?

Reply:
Great catch, I think originally I was planning on using them, so I added them but then forgot about it when i was trying to fix the warnings. I'll remove that.